apps/opik-backend/config.yml

```diff
@@ -69,7 +69,13 @@ server:
     enabled: true
 
 rateLimit:
-    enabled: ${RATE_LIMIT_ENABLED:-false}
-    generalLimit:
-      limit: ${RATE_LIMIT_GENERAL_EVENTS_LIMIT:-10000}
-      durationInSeconds: ${RATE_LIMIT_GENERAL_EVENTS_DURATION_IN_SEC:-60}
+  enabled: ${RATE_LIMIT_ENABLED:-false}
+  generalLimit:
+    limit: ${RATE_LIMIT_GENERAL_EVENTS_LIMIT:-10000}
+    durationInSeconds: ${RATE_LIMIT_GENERAL_EVENTS_DURATION_IN_SEC:-60}
+
+metadata:
+  version: ${OPIK_VERSION:-latest}
+  usageReport:
+    enabled: ${OPIK_REPORTING_ENABLED:-true}
+    url: ${OPIK_REPORTING_URL:-https://stats.comet.com/notify/event/}
```

Comment:
I don't think the `usageReport` fits under the `metadata`. Additionally, the new env var don't follow the same naming as the configuration keys. e.g: usageReport.enabled vs `OPIK_REPORTING_ENABLED`. The `OPIK` prefix is good, but the rest should be as similar as possible.